tests/tests_suite_test.go

```diff
@@ -166,4 +175,13 @@ func BuildTestSuite() {
 			Expect(err).ToNot(HaveOccurred())
 		}
 	})
+
+	JustAfterEach(func() {
```

Comment:
This will make sure we get artifacts that got deleted in AfterEach blocks
/cc @alromeros